### DISCUSSION TITLE
人工智慧AI能預測股市？

### DISCUSSION THREAD

**Discussion by MingWei174** [1101-GS4719-C/Python-Community-GS4719-C#2051] [General]
https://tw.stock.yahoo.com/news/%E4%BA%BA%E5%B7%A5%E6%99%BA%E6%85%A7%E8%83%BD%E9%A0%90%E6%B8%AC%E8%82%A1%E5%B8%82-%E4%BB%96%E6%9B%9Dai%E4%BA%92%E6%AF%86%E6%9C%83-%E5%AE%8C%E5%85%A8%E7%9F%9B%E7%9B%BE-%E7%B6%B2%E6%8E%80%E5%85%A9%E6%B4%BE%E8%AB%96%E6%88%B0-030600826.html
對於未來會不會發展出「預測股市的AI」這個問題，在網路上掀起了兩派說法，有一部分人認為AI預測股市不可行，主要是因為「股市無法預測，才會是股市，因為操作股市的人，是人，人性是無法預測」，但另外一部分的人卻認為AI預測股市早已存在，近期的股市閃崩就是各家機構程式和也許AI輔助的交易導致的。
我自己則是認為如果未來發展出預測股市的AI且普及性非常高，人人皆可以輕鬆預測未來股市走向，將會導致沒有行情，全球股市會失去意義，因為變成了「100%效率市場」，甚至之後可能會變成金融機構之間的機器人AI大戰，參與投資的散戶們也無從干涉。

**Comment by hunteryang0108** [comment]
我認為股票交易本身就沒有絕對的獲利方法，AI預測頂多只能增加獲利的機率，並不能代表使用AI就一定能夠獲利。不過只要有利可圖，就會有人投出相關的產業，因此我認為開發股票投資預測的AI機器人是一個不錯的選擇。

**Comment by Jerryguo888** [comment]
未來股市如何演變將受到AI的影響，而AI在預測股市方面的發展可能會帶來諸多挑戰和機遇。儘管AI技術有能力可以改變市場格局，但市場的基本性質和人性因素仍然是不可忽視的，投資者和市場參與者需要靈活應對技術變革，並且考慮如何在這樣的環境下繼續保持競爭力和適應性。
